src/libcaf.h

```diff
@@ -313,4 +314,10 @@ void PREFIX (unlock) (caf_token_t, size_t, int, int *, char *, int);
 void PREFIX (event_post) (caf_token_t, size_t, int, int *, char *, int);
 void PREFIX (event_wait) (caf_token_t, size_t, int, int *, char *, int);
 void PREFIX (event_query) (caf_token_t, size_t, int, int *, int *);
+
+/* Language extension */
+#ifdef HAVE_MPI
+MPI_Fint PREFIX (get_communicator) (caf_team_t *);
```

Comment:
yes it is an extension, for work with some folks at NCAR. Essentially they have some MPI codes that they launch in large parallel batches, but they want to migrate to coarrays incrementally, and also, the codes have some shared initialization costs. So the idea is to use teams to parallelize the embarrassingly parallel batch jobs (instead of shell scripts/python etc.) and also perform common initialization. I agree that this needs some better documentation, and I like the idea of `libcaf-ext.h`